### PR TITLE
Display multiple motds in CLI and deactivate old ones

### PR DESCRIPTION
**Pull request by valyo**
Adds ls and deactivate subcommands to `dds motd`
and displaying of multiple active motds
works with https://github.com/ScilifelabDataCentre/dds_web/pull/1212

Please include the following in this section

- [x] Summary of the changes and the related issue
- [x] Relevant motivation and context
- [ ] Any dependencies that are required for this change

Fixes DDS-1305

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

Please delete options that are not relevant.

- [ ] Any dependent changes have been merged and published in downstream modules
- [x] Rebase/merge the branch which this PR is made to
- [ ] Product Owner / Scrum Master: This PR is made to the `master` branch and I have updated the [version](../dds_cli/version.py)

## Formatting and documentation

- [x] I have added a row in the [changelog](../CHANGELOG.md)
- [x] The code follows the style guidelines of this project: Black / Prettier formatting
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Tests

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: dds_cli/motd_manager.py [hunk 1]
```diff
@@ -73,15 +73,52 @@ def add_new_motd(self, message):
...
         else:
-            return response_json.get("message")
+            # Get items from response
+            motd = response.get("motds")
+            if motd:
+                motds, keys = dds_cli.utils.get_required_in_response(
+                    keys=["motds", "keys"], response=response
+                )
+            else:
+                LOG.info("No active Message Of The Day found")
```
Comment (inaod568) [inline dds_cli/motd_manager.py:94]:
```suggestion
            # Get items from response
            motd = response.get("motds")
            if not motd:
                message = response.get("message")
                if not message:
                    LOG.error("No motds or info message returned from API.")
                
                LOG.info(message)

            motds, keys = dds_cli.utils.get_required_in_response(
                keys=["motds", "keys"], response=response
            )
```

**Review by inaod568 — CHANGES_REQUESTED** [review]
This is to match the suggested change in the web part

**Comment by inaod568** [comment]
@valyo Ok to merge this too? I altered it slightly in the beginning of the week.

**Comment by valyo** [comment]
> @valyo Ok to merge this too? I altered it slightly in the beginning of the week.

sure, I already saw the changes

**Review by inaod568 — APPROVED** [review]

**Pull request merged**